index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Html Form</title>
+    <link rel="stylesheet" href="assets/css/styles.css"/>
+  </head>
+  <body>
+    <div class="top_right">
+      <div class="flex_container">
+        <p>Already have an Intuit account?</p>
+        <a href="https://accounts.intuit.com/account-recovery.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS">I forgot my User ID or Password</a>
+      </div>
+      <button class="button" type="button" name="button"><img src="assets/img/sign_in.png" alt="Not found"/></button>
+    </div>
+
+    <main class="center">
+      <div class="width">
+        <figure class="center">
+          <img src="assets/img/header.png" alt="Not found"/>
+        </figure>
+
+<!-- form -->
+        <form class="form" method="POST">
+          <div class="center_form">
+            <div class="center_text">
+              <h2>Create an Intuit account</h2>
+              <div class="center_text">
+                <h3>One account for everything Intuit, including Mint. </h3>
+                <a href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account">Learn More.</a>
+              </div>
+            </div>
+
+            <div>
+
+
+              <div class="center_inputs">
+                <label>Email Adress</label>
```

Comment:
`SUGGESTION`

Consider associating the `<label>` with the `<input>` element, to do this you need to give the `<input>` an id attribute. The `<label>` then needs a for attribute whose value is the same as the input's id. 
- Example 1: 

``` html
  <label for="email">Email address</label> <br>
  <input type="email" name="email" id="email">
```

- Example 2: Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screenreader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)